docs/configuration/logging.md

```diff
@@ -0,0 +1,53 @@
+---
+description: Configure log levels and log retention periods for your Codacy Self-hosted instance.
+---
+
+# Logging
+
+Currently, Codacy Self-hosted supports two types of log configurations:
+
+-   **[Log level](#setting-up-log-levels):** The severity of the messages that will be shown in logs. We recommend that you don't use a log level lower than WARN, as the logs are useful to troubleshoot any issues in your Codacy Self-hosted instance.
+-   **[Log retention period](#configuring-log-retention-period):** The period during which logs will be stored, before being removed by the cleanup job specified in the configuration.
+
+The sections below provide instructions on how to configure each logging configuration.
+
+## Setting up log level
+
+The log level of most components can be configured by editing the value of `COMPONENT.config.logLevel` in the `values-production.yaml` file that is used to install Codacy, like is shown in the example below:
+```yaml
+worker-manager:
+  replicaCount: 2
+  config:
+    logLevel: WARN
+  resources:
+    limits:
+      cpu: 500m
+      memory: 1000Mi
+    requests:
+      cpu: 200m
+      memory: 200Mi
+```
+The components that currently support this configuration are `codacy-api`, `listener`, `engine`, and `worker-manager`.
+
+The list of supported values for this configuration is shown below. Note that each level also prints the information of the levels higher than it:
+
+-   DEBUG: **Most** verbose level. Logs detailed information on the flow through the system.
+-   INFO: Logs interesting events such as application startup and shutdown, important events on the flow of the system, or behavior that is skipped for expected reasons.
+-  WARN: Logs runtime events that are unexpected, or undesirable, but not necessarily wrong such as the use of deprecated APIs.
+-  ERROR: **Least** verbose level. Only logs runtime errors, or unexpected conditions that prevent the expected behavior from happening.
```

Comment:
There were a couple of missing spaces in my previous suggestion, sorry! :sweat_smile: 

```suggestion
-   DEBUG: **Most** verbose level. Logs detailed information on the flow through the system.
-   INFO: Logs interesting events such as application startup and shutdown, important events on the flow of the system, or behavior that is skipped for expected reasons.
-   WARN: Logs runtime events that are unexpected, or undesirable, but not necessarily wrong such as the use of deprecated APIs.
-   ERROR: **Least** verbose level. Only logs runtime errors, or unexpected conditions that prevent the expected behavior from happening.
```

Reply:
@prcr @lventura-codacy My only suggestion is that we let the reader know what the default log level is, if possible. Is the default the same for each component?

Reply:
@dan-codacy, we now mention that DEBUG is the default log level (it's the same for all components).

Could you please have another look at the page? You can see the Markdown preview if that helps:

https://github.com/codacy/chart/blob/log-configuration-doc/docs/configuration/logging.md